### DISCUSSION TITLE
Add model repositories for our Ruxolitinib and Posaconazole model?

### DISCUSSION THREAD

**Discussion by OliversLab** [Open-Systems-Pharmacology/Forum#1774] [Models]
Hi there,

two years ago, we published PBPK models of Ruxolitinib and Posaconazole for a DDI via CYP3A4 where Ruxolitinib is the victim drug.

repository topics could be: pbpk and ddi

Ruxolitinib-model
and 
Posaconazole-model

might be suitable names.

Is it possible to fork them from my repository or only the other way around?

Thank you so much and keep up the awesome work!

best,
Oliver

**Comment by PavelBal** [comment]
Model repositories within OSP should be the "original" repos and not forked. So you would have to create new repos here and upload the models.

@Yuri05 Do we have a guide/documentation for creating model repos?

**Reply by Yuri05** [reply]
>Model repositories within OSP should be the "original" repos and not forked. So you would have to create new repos here and upload the models.

This 🔼 

>Do we have a guide/documentation for creating model repos?

yes. linked on the landing page 😉 
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/0869c380-7672-4da1-975d-f353b9bc925a)

**Comment by OliversLab** [comment]
Thank you so much, I had a look at the guide that's why I asked here ;-) 
I cannot create a repository by myself ;-)

Thank you!
best
Oliver

**Reply by Yuri05** [reply]
Ok, got it now :)
So here are the repositories:
https://github.com/Open-Systems-Pharmacology/Ruxolitinib-Model
https://github.com/Open-Systems-Pharmacology/Posaconazole-Model
